lib/containers/common.pm

```diff
@@ -134,9 +134,10 @@ sub clean_container_host {
     if ($runtime =~ /buildah/) {
         assert_script_run("$runtime rm --all");
         assert_script_run("$runtime rmi --all --force");
-    } else {
-        assert_script_run("$runtime stop \$($runtime ps -q)", 180) if script_output("$runtime ps -q | wc -l") != '0';
-        assert_script_run("$runtime system prune -a -f",      180);
+    }
+    else {
```

Comment:
```suggestion
    } else {
```

Reply:
This came up from the rebase. to be honest i dont know what to do with such comments as there is not a code convention and each one prefers a different style.

Reply:
but you should try to keep the same as already exists in the same file https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/11878/files#diff-416f30e165941aab0be675b0f6760989acb6bb5ccd4ee32d1ee42d52dae71e6dR125

Reply:
restored 39cd34e84. but i think we have to discuss this because it feels that we force specific coding style here.

Reply:
sure you can follow any style which you like I don't care , but I am strongly against mixing all of them at least in one file